package.json

```diff
@@ -17,6 +17,7 @@
     "@types/react": "^18.2.0",
     "@types/styled-components": "^5.1.26",
     "babel-loader": "^8.2.2",
+    "lodash": "^4.17.21",
```

Comment:
why we need lodash?

the bundle is too heavy to use just one function from this, in many projects we create our own version of debounce becouse we can't use library in a bundle just for some strings of code, so in this case it would be better to create utility folder and create debounce function by ourselves
![image](https://github.com/solovevayaroslavna/simbirgwent/assets/90199600/b3cd381f-a236-4b34-9e96-b9c96f91a322)

https://learn.javascript.ru/task/debounce - it is useful to know how to write your own debounce also because this task occurs at interviews in the section lifecoding

Reply:
It turns out that this library is not used in the code and was loaded for an earlier version of the code. Therefore, the unused library has been removed.